src/sdcard.cpp

```diff
@@ -93,3 +95,21 @@ SDCard::SdCardDirectory SDCard::openDir(const String &pathName)
 
     return result;
 }
+
+SDCard& SDCard::load()
+{
+    if (loadedSDCard_ == nullptr)
+    {
+	loadedSDCard_ = new SDCard;
+    }
+    return *loadedSDCard_;
+}
+
+void SDCard::unload()
+{
```

Comment:
Is this realy needed? We can't exchange the SD Card without unplugging the power sooooo.....

Reply:
Got removed.